### DISCUSSION TITLE
Sentry @ KubeCon + CloudNativeCon Amsterdam 2023

### DISCUSSION THREAD

**Discussion by smeubank** [getsentry/sentry#46938] [Solution Feedback]
Sentry is sponsoring a booth @ [KubeCon + CloudNativeCon Amsterdam 2023](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/) 18th-21st of April.

Recently this year we shipped OpenTelemetry support https://github.com/getsentry/sentry/discussions/40712. We're excited to show off Sentry, talk with users, and hear back what people think about our recent OTel support and any and all new features.

Is there anything else you would like to hear from us at the booth? 
Are you interested to see support for any other Cloud-Native/CNCF projects?

Looking forward to seeing you at the booth!

**Comment by bruno-garcia** [comment]
Let us know how often dotnet comes up. Curious how big it is in this event.
